hydra-works.gemspec

```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'hydra-pcdm', '>= 0.8'
   spec.add_dependency 'hydra-derivatives', '~> 3.0'
   spec.add_dependency 'hydra-file_characterization', '~> 0.3', '>= 0.3.3'
+  spec.add_dependency 'active-fedora', '~> 11.0.0.rc7'
```

Comment:
I required AF 11 so I could reproduce the original missing constant AF::OmDatastream error, but a datastream-less implementation should work fine under AF 10.